recogym/envs/abstract.py

```diff
@@ -309,17 +312,13 @@ def _store_bandit(action, reward):
                 _store_bandit(action, reward)
 
             _store_organic(observation)
```

Comment:
Maybe remove this?

Reply:
reserve in case that 'prob_leave_organic' is not zero.

Reply:
I don't get it. What's the point to keep this if it is not followed by any bandit trials?

Reply:
I think removing this implicitly will lead to unsafe behaviors as it's a base class. Otherwise, for other setting with non-zero prob_leave_organic, it may cause bugs hard to track. 

We should add 'assert prob_leave_organic == 0' in this file if removing it is need.